src/System.Private.CoreLib/src/System/Runtime/ProfileOptimization.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file should be removed from this change.

Reply:
You mean that `System.Runtime.Extensions.dll` produced by corefx will work on corert runtime?(sorry for this trivial question...but i don't know well how corert works today :confounded:).

Reply:
> We try to avoid situations where it is in CoreLib for one runtime and CoreFX for the other runtime.

Ok understood thank's a lot for patience!